app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,89 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.entity.Product;
+import com.codesoom.assignment.exception.ProductNotFoundException;
+import com.codesoom.assignment.repository.ProductRepository;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 새로운 상품을 생성한다.
+     *
+     * @param product the product
+     * @return 저장한 상품
+     */
+    public Product create(Product product) {
+       return productRepository.save(product);
+    }
+
+    /**
+     * id로 product 객체를 찾아 리턴한다
```

Comment:
JavaDoc의 메인 디스크립션은 타인에게 공개되는 문장입니다. 마침표를 잊지 마세요.

```suggestion
     * 상품을 찾아 리턴한다.
```